src/gam/gapi/directory/printers.py

```diff
@@ -76,15 +58,32 @@ def create():
 
 
 def delete():
```

Comment:
We could also use a helper function to convert user supplied displayName for printer to the printer ID. Feel free to take a crack at it. I'm pretty sure displayName does not have to be unique for printers though so if we get more than one match we'll need to error out with a list of IDs that match and whatever info we know about them so admin can be more specific.